project/sites/hiaredressni/themes/hiaredressni_theme/src/scss/components.scss

```diff
@@ -25,3 +25,33 @@
 }
 
 
+h1 {
```

Comment:
You shouldn't need to add this for the H1, you may need to copy the init/_typography.scss from another site. e.g. mentalhealthchampion to get all the new heading sizes.

Reply:
Thanks Adam, I have removed this code and updated the typography